src/XamlStyler.dotUltimate/gradle.properties

```diff
@@ -0,0 +1,15 @@
+# Any property can be overwritten from command-line via
+#   -P<propertyName>=<value>
+
+DotnetPluginId=ReSharperPlugin.XamlStyler.dotUltimate
+DotnetSolution=XamlStyler.dotUltimate.sln
+RiderPluginId=xamlstyler.dotultimate
+PluginVersion=0.0.1337
+
+BuildConfiguration=Release
+
+# Possible values:
+#    2019.2-SNAPSHOT
+#    2019.2-EAP2-SNAPSHOT
+#    2019.2
+ProductVersion=2020.1.0
```

Comment:
This is the Rider version to build against. Right now, this means Rider 2020.1 and up, < 2020.2.

Reply:
Will need updating for newer Rider versions. A plugin across multiple versions (e.g. 2020.1 and 2020.2) is possible, but hard to achieve.

Reply:
Typically 3 releases per year.

Reply:
Will annotate "things that will need updating" with a Rider release with the comment:

`// RIDER:` throughout the codebase.

Next commit will have these.